cheroot/makefile.py

```diff
@@ -32,6 +35,14 @@ def _flush_unlocked(self):
                 n = self.raw.write(bytes(self._write_buf))
             except io.BlockingIOError as e:
                 n = e.characters_written
+            except (
+                SSL.WantReadError,
+                SSL.WantWriteError,
+                SSL.WantX509LookupError,
+            ):
+                # these errors require retries with the same data
+                # if some data has already been written
+                n = 0
```

Comment:
perhaps, just skip that `del` altogether?
```suggestion
                continue
```

Reply:
Agreed